src/review/views/review_view.py

```diff
@@ -29,11 +31,19 @@ class ReviewViewSet(viewsets.ModelViewSet, ReactionViewActionMixin):
     order_fields = "__all__"
     queryset = Review.objects.all()
     ordering = ("-created_date",)
+    ALLOWED_CONTENT_TYPES = "rhcommentmodel"
 
     def create(self, request, *args, **kwargs):
         unified_document = ResearchhubUnifiedDocument.objects.get(id=args[0])
         request.data["created_by"] = request.user.id
         request.data["unified_document"] = unified_document.id
+
+        if request.data.get("content_type") not in ReviewViewSet.ALLOWED_CONTENT_TYPES:
```

Comment:
```suggestion
        if request.data.get("content_type") not in self.ALLOWED_CONTENT_TYPES:
```
Also `ALLOWED_CONTENT_TYPES` needs to be in a list